ui/components/DashboardComponent.js

```diff
@@ -945,6 +952,29 @@ class DashboardComponent extends React.Component {
       return <>Running latest Meshery version.</>;
     };
 
+    /**
+     * openReleaseNotesInNew returns the appropriate link to the release note
+     * based on the meshery's current running channel and version.
+     *
+     * @returns {React.ReactNode} react component to display
+     */
+    const openReleaseNotesInNew = () => {
+      const { release_channel, build } = this.state.versionDetail;
+
+      if (release_channel === "edge")
+        return (
+          <Link href="https://docs.meshery.io/project/releases" target="_blank">
+            <OpenInNewIcon fontSize="small" />
+          </Link>
+        );
+
+      return (
+        <Link href={`https://docs.meshery.io/project/releases/${build}`} target="_blank">
+          <OpenInNewIcon fontSize="small" />
```

Comment:
Thank you for this, @CipherGirl. 👍 

This styling leaves the icon size unresponsive. 

View: 100%
<img width="659" alt="Screen Shot 2021-08-10 at 1 19 00 PM" src="https://user-images.githubusercontent.com/7570704/128914212-f49aeb34-e4e7-4380-9b66-5c31b532e0b0.png">
Will you reduce the icon size to half of this height?


View: 200%
<img width="638" alt="Screen Shot 2021-08-10 at 1 19 07 PM" src="https://user-images.githubusercontent.com/7570704/128914246-46fc1b85-80e3-4f51-954e-e3c1da80b360.png">

Reply:
I didn't have this issue in my browser. But still changed the styling into CSS format hope this will work.. Let me know :)